src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
 public class WorkWithFile {
+    static final String SEPARATOR = ",";
+    static final int DESCRIPTION_OF_THE_LIST = 0;
+    static final int VALUE_OF_THE_LIST = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(makeReport(readFromFile(fromFileName)),toFileName);
+    }
+
+    private String[] readFromFile(String fromFileName) {
+        ArrayList<String> list = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String value;
+            while ((value = bufferedReader.readLine()) != null) {
+                list.add(value);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Cant read from file" + fromFileName, e);
+        }
+        return list.toArray(new String[0]);
```

Comment:
Why you use list and make array from this? You can read from file and save in just list or better way save it in String